app/src/main/res/values-en/strings-en.xml

```diff
@@ -21,6 +21,7 @@
     <string name="scenario3_explanation">Add a switch to the circuit, so you can flick the Lamp on and off. Afterwards press the switch to turn the lamp on.</string>
     <string name="scenario4_explanation">Now you understand all components, make your own circuit with 2 lightbulbs and a way to turn the lights off and on.</string>
     <string name="scenario5_explanation">You now know how to build a circuit, time to try it out yourself! Make circuit with a switch and a lamp, and ensure that the lamp turns on.</string>
+    <string name="scenario6_explanation">You can also split wires, in this way you can make 2 circuits with 1 battery! Finish this circuit by adding a lamp and a switch on the side where it is needed.</string>
```

Comment:
Je hebt het over een kant waar het nodig is, maar volgens mij is het gewoon één van de kanten.